include/awkward/gpu-kernels/identities.h

```diff
@@ -0,0 +1,20 @@
+//
+// Created by root on 5/14/20.
+//
```

Comment:
New files have a URL to the license, not a creation date. (That information is all in git.)